docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+services:
+  app:
+    depends_on:
+      - mongodb
+    build: .
```

Comment:
Could you change this from ``build: .`` to ``image: zyner/xyter:2022.07.0`` for next release.

Reply:
Yes, but you shouldn't forget to push the image before releasing.

Reply:
True! Make it ``image: zyner/xyter:latest`` instead. This way I can automate Docker Image Build via a Github Action. 👍

Reply:
You can look at the build_image step [here](https://github.com/Unately/uBingo/blob/master/.github/workflows/release.yml) I've done something like that already myself.

Reply:
Would be super cool if you could make a PR with that too 😅 To push dev versions and production versions, like tags as ``2022.07.0`` or ``dev-2022.07.0`` you get the point, with images as the branch name.